pkg/csv2lp/csv2lp.go

```diff
@@ -144,6 +145,17 @@ func (state *CsvToLineReader) Read(p []byte) (n int, err error) {
 			state.buffer = append(state.buffer, '\n')
 			break
 		} else {
+			// detect non-standard line endings in header lines #106
+			for _, col := range row {
+				if idx := strings.Index(col, "\r"); idx >= 0 && idx+1 < len(col) && col[idx+1] != '\n' {
+					log.Println(
+						fmt.Sprintf("WARNING: %v. Neither CRLF nor LF line endings?",
```

Comment:
Suggestion on the wording:
```suggestion
						fmt.Sprintf("WARNING: %v. Only CRLF and LF line endings are supported.",
```